src/Filament/Widgets/SiteStatsWidget.php

```diff
@@ -16,7 +16,9 @@ protected function getViewData(): array
     {
         return [
             'sites' => SiteRepository::query()
-                ->with(['uptimeCheck', 'sslCertificateCheck'])
+                ->with(['uptimeCheck', 'sslCertificateCheck', 'systemMetrics' => function ($query) {
+                    $query->latest();
+                }])
```

Comment:
```suggestion
                return [
            'sites' => SiteRepository::query()
                ->with(['uptimeCheck', 'sslCertificateCheck', 'systemMetrics'])
                ->withCount(['exceptionLogs' => function ($query) {
                    $query->where('status', 'unresolved');
                }])
                ->get(),
        ];
```

Reply:
@rpereira-tae and how can I access the values in the view?

`$cpuUsage = $site->systemMetrics->first()->cpu_usage ?? null;`

Reply:
Sorry this change is not needed, it can remain the way you requested it. I thought the `$query->latest()` was being ignored